xapian-letor/include/xapian-letor/ranker.h

```diff
@@ -37,36 +37,57 @@
-     *  @param  qrel_file       Here supply the path to the qrel file (in free text form) containing the relevance judgements for the
-     *          queries in the training file. This file should be in standard format specified.
-     *  @param  msetsize   This is the mset size used for the first retrieval for training queries. It should be selected depending on the qrel file
-     *          and database size.
-     *  @param  filename   Filename path where the training file has to be stored. Default is "./training-data.txt".
-     *  @param  flist      Xapian::FeatureList object definining what set of features to use for preparing the training file.
-     *          It is initialised by DEFAULT set of Features by default.
-     *          To use a custom set of features, pass a customised Xapian::FeatureList object.
-     *  @exception FileNotFoundError will be thrown if file not found at supplied path
-     *  @exception LetorParseError will be thrown if query file or qrel file could not be parsed
+     *  @param  query_file     Here you have to give a path to the file
+     *				(in free text form) containing training queries
+     *				in specified format.
+     *  @param  qrel_file      Here supply the path to the qrel file
+     *				(in free text form) containing the relevance
+     *				judgements for the queries in the training
+     *				file. This file should be in standard format
+     *				specified.
+     *  @param  msetsize	This is the mset size used for the first
+     *				retrieval for training queries.
+     *				It should be selected depending on the qrel
+     *				file and database size.
+     *  @param  filename	Filename path where the training file has to
+     *				be stored. Default is "./training-data.txt".
+     *  @param  flist		Xapian::FeatureList object definining what set
+     *				of features to use for preparing the training
+     *				file. It is initialised by DEFAULT set of
+     *				Features by default. To use a custom set of
+     *				features, pass a customised Xapian::FeatureList
+     *				object.
+     *
+     *  @exception FileNotFoundError will be thrown if file not found at
+     *		    supplied path
+     *  @exception LetorParseError will be thrown if query file or qrel file
+     *		    could not be parsed
      */
     XAPIAN_VISIBILITY_DEFAULT
-    void prepare_training_file(const std::string & db_path,
-			       const std::string & query_file,
-			       const std::string & qrel_file,
-			       Xapian::doccount msetsize,
-			       const std::string & filename = "training-data.txt",
-			       const Xapian::FeatureList & flist = Xapian::FeatureList());
+    void
+    prepare_training_file(const std::string & db_path,
+			   const std::string & query_file,
+			   const std::string & qrel_file,
+			   Xapian::doccount msetsize,
+			   const std::string & filename = "training-data.txt",
+			   const Xapian::FeatureList & flist =
+			   Xapian::FeatureList());
```

Comment:
I'd take advantage of us being inside `namespace Xapian {...}` here and drop the `Xapian::` from `Xapian::FeatureList` here so this line doesn't need wrapping.

In a case which really needed wrapping, the default value ought to be indented for clarity.